kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTab.js

```diff
@@ -38,9 +38,11 @@ define(['bluebird', 'common/html', './jobStatusTable'], (Promise, html, JobStatu
         }
 
         function stop() {
-            container.innerHTML = '';
             if (jobStatusTableWidget) {
-                return jobStatusTableWidget.stop();
+                jobStatusTableWidget.stop();
+            }
+            if (container) {
+                container.innerHTML = '';
```

Comment:
stop the job status table before emptying the container (wasn't a bug, just doing things in a more logical order)